pyrene/src/components/DateTimePicker/DateTimePicker.tsx

```diff
@@ -0,0 +1,290 @@
+import React, {
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+
+import ReactDPWrapper from './ReactDatePickerWrapper/ReactDatePickerWrapper';
+import DateTimeInput from './DateTimeInput/DateTimeInput';
+
+import {
+  getFutureDate, standardEUDateFormat, standardEUTimeFormat,
+  isValidDate, isValidTime, convertToUTCtime, convertToZoneTime,
+  convertDateTypeToString, convertTimeTypeToString, dateTypeToStandardEUDateFormat,
+  getDateTypeFromddmmyyyyWithSep, getTimeTypeFromhhmmWithSep, getErrors,
+  errorDateBool, errorTimeBool,
+} from '../../utils/DateUtils';
+
+
+type OnFunction = (value?: number) => void;
+
+export interface DateTimePickerProps{
+  calendarOpened?: boolean,
+  closeOnSelect?: boolean,
+  customCalendar?: (props:{
+    children: React.ReactNode[]
+  }) => React.ReactNode,
+  /**
+   * Boolean to toggle time display
+   */
+  dateOnly?: boolean,
+  /**
+   * This is a Date object that represents the end date of the component
+   */
+  endDate?: Date,
+  inline?: boolean,
+  /**
+   * This is a string that represents the label of the component
+   */
+  label?: string,
+  /**
+   * This is a timestamp that represents the maximum date allowed by the component
+   */
+  maxDateTime?: number,
+  /**
+   * This is a timestamp that represents the minimum date allowed by the component
+   */
+  minDateTime?: number,
+  /**
+   * Name that can be used to uniquely identify the component
+   */
+  name?: string,
+  /**
+   * Function to handle onBlur event
+   */
+  onBlur?: () => OnFunction,
+  onCalendarOpen?: () => void,
+  onClickOutside?(event: React.MouseEvent<HTMLDivElement>): void,
+  /**
+  * Function to handle onChange event
+  */
+  onChange: OnFunction,
+  /**
+   * This is a string array that represents the start and end labels of the component
+   */
+  range?: boolean,
+  /**
+   * Boolean to indicate if this input selects the end date of a range
+   */
+  selectEnd?: boolean,
+  /**
+  * Boolean to indicate if this input selects the start date of a range
+  */
+  selectStart?: boolean,
+  /**
+  * This is a Date object that represents the start date of the component
+  */
+  startDate?: Date,
+  tabNum?: number,
+  /**
+   * This is a unix timestamp, which is the number of seconds that have elapsed since Unix epoch
+   */
+  timeStamp?: number | null
+  /**
+   * This is must be a IANA time zone string
+   */
+  timeZone?: string,
+}
+
+const DateTimePicker: React.FC<DateTimePickerProps> = ({
+  calendarOpened,
+  closeOnSelect = true,
+  customCalendar,
+  dateOnly = false,
+  endDate,
+  inline = false,
+  label,
+  maxDateTime = getFutureDate({ years: 1 }),
+  minDateTime = 0,
+  name,
+  onCalendarOpen,
+  onChange,
+  onClickOutside,
+  range = false,
+  selectEnd,
+  selectStart,
+  startDate,
+  tabNum,
+  timeStamp,
+  timeZone = 'Europe/Zurich',
```

Comment:
Use the [default](https://stackoverflow.com/questions/6939685/get-client-time-zone-from-browser) time zone of user's web client.